sources/exception.asciidoc

```diff
@@ -141,6 +141,33 @@ These exception cases can be determined at compile time by a tool such as an ann
 These exception cases can be determined at compile time by a tool such as an annotation processor.
 ====
 
+[[exception-valueextractordefinition]]
```

Comment:
I think the exception chapter should say that when no VE is found on a constrained container element, exception X is raised.
Same for a container type that leads to two maximizing stuff VE